content-src/lib/selectLayoutRender.js

```diff
@@ -139,19 +139,29 @@ export const selectLayoutRender = (state, prefs, rickRollCache) => {
     return { ...component, data };
   };
 
+  const filterComponent = (c) => {
```

Comment:
Changes in this file are removing any components or collections that are sponsored if spocs are disabled.